src/main/java/org/jenkinsci/plugins/webhookstep/RegisterWebhookStep.java

```diff
@@ -8,16 +8,30 @@
 
 import hudson.Extension;
 
+import java.io.UnsupportedEncodingException;
+import java.net.URLEncoder;
+
 public class RegisterWebhookStep extends Step {
 
+    final String token;
+
     @DataBoundConstructor
     public RegisterWebhookStep() {
+        this.token = null;
+    }
 
+    public RegisterWebhookStep(String token) throws UnsupportedEncodingException {
```

Comment:
It would be better to do this with a @DataBoundSetter instead of a new constructor.

Reply:
Although I didn't like the idea of making token field non-final, anyway see new commit [ed6535d](https://github.com/jenkinsci/webhook-step-plugin/pull/12/commits/ed6535d8d9dd9624edbc210335d86beb9e7c41cc).